.travis.yml

```diff
@@ -55,7 +55,6 @@ matrix:
 
     # todo update to default 3.6 (aka removing the line with PYTHON_VERSION here)
     - env: NAME="Lint"
-           PYTHON_VERSION=2.7
            MAIN_CMD="pylint --rcfile=package/.pylintrc package/MDAnalysis && pylint --rcfile=package/.pylintrc testsuite/MDAnalysisTests"
```

Comment:
`pylint` reads the `PYLINTRC` env variable for the location of the rcfile. I vote for setting that and skipping the `--rcfile` flag; should make the `MAIN_CMD` tidier and more readable.